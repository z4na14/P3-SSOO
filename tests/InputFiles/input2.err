5 8 15 2 4 6 8 10 1 3 5 7 9 2 4 6 8 10
