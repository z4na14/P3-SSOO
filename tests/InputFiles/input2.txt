4 5 5 2 1 2 3 3 5 2