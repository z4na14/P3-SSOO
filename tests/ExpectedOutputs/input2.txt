[OK][factory_manager] Process_manager with id 5 has been created.
[OK][factory_manager] Process_manager with id 1 has been created.
[OK][factory_manager] Process_manager with id 3 has been created.
[OK][process_manager] Process_manager with id 5 waiting to produce 2 elements.
[OK][process_manager] Process_manager with id 1 waiting to produce 3 elements.
[OK][process_manager] Process_manager with id 3 waiting to produce 2 elements.
[OK][process_manager] Belt with id 5 has been created with a maximum of 5 elements.
[OK][process_manager] Belt with id 1 has been created with a maximum of 2 elements.
[OK][process_manager] Belt with id 3 has been created with a maximum of 5 elements.
[OK][queue] Introduced element with id 0 in belt 5.
[OK][queue] Introduced element with id 1 in belt 5.
[OK][queue] Introduced element with id 0 in belt 1.
[OK][queue] Introduced element with id 1 in belt 1.
[OK][queue] Introduced element with id 2 in belt 1.
[OK][queue] Introduced element with id 0 in belt 3.
[OK][queue] Introduced element with id 1 in belt 3.
[OK][queue] Obtained element with id 0 in belt 5.
[OK][queue] Obtained element with id 1 in belt 5.
[OK][queue] Obtained element with id 0 in belt 1.
[OK][queue] Obtained element with id 1 in belt 1.
[OK][queue] Obtained element with id 2 in belt 1.
[OK][queue] Obtained element with id 0 in belt 3.
[OK][queue] Obtained element with id 1 in belt 3.
[OK][process_manager] Process_manager with id 5 has produced 2 elements.
[OK][process_manager] Process_manager with id 1 has produced 3 elements.
[OK][process_manager] Process_manager with id 3 has produced 2 elements.
[OK][factory_manager] Process_manager with id 5 has finished.
[OK][factory_manager] Process_manager with id 1 has finished.
[OK][factory_manager] Process_manager with id 3 has finished.
[OK][factory_manager] Finishing.
